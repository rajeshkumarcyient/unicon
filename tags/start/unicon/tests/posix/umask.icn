#

procedure main()
   write(image(umask("rwxr-xr-x")))
   write(image(umask()))
end
