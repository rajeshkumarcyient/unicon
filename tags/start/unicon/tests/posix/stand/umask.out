"rwxr-xr-x"
"rwxr-xr-x"
